src/calendar/index.njk

```diff
@@ -2,44 +2,121 @@
 title: Calendar
 layout: layouts/base.njk
 ---
-<div class="grid gap-4">
-  <h2>Events this week</h2>
 
+<div class="grid grid-flow-row gap-3">
+  <h2 class="text-xl mt-5 mx-4">EVENTS THIS WEEK</h2>
+
+  {% set datevariable = "null" %}
   {% for event in events | eventThisWeek | eventsSort %}
-    <a
-      href="/calendar/{{ event.startDate | formatDate}}/{{ event.slug }}/"
-      class="block border p-4 hover:border-blue"
-      itemscope itemtype="https://schema.org/Event"
-    >
-      <date class="block text-xs" itemprop="startDate" content="{{ event.startDate }}">
-        {{ event.startDate | formatDate('dddd, D MMMM YYYY, h:mma') }} ({{ event.duration }})
-      </date>
 
-      <p itemprop="name">
-        {{ event.title }}
-      </p>
+    {% if event.startDate != datevariable %}
+      <!-- DATE OF EVENT-->
+      <div class="mx-4 text-lg text-black">
+        {{ event.startDate | formatDate('dddd D MMMM') }}
+      </div>
+      <!-- LINE -->
+      <hr class="mx-4 -mt-1 text-black"/>
+      {% set datevariable = event.startDate %}
+    {% endif %}
+
+    <div class="mx-4 grid grid-flow-row grid-cols-2 gap-3">
```

Comment:
This isn't doing what I think you think it is doing; it's not creating the grid yet. Can we have `gap-4` to keep things standard across the rest of the site.